src/wacks4shop/wack.py

```diff
@@ -21,25 +21,24 @@
 def main(db: Wackabase, dry=False):
     try:
         logger.info("TIME TO WACK")
-        logger.info("Dry run: %s", dry)
```

Comment:
why get rid of this log?

Reply:
I thought it was noisy. I'll change it to `logger.debug`